packages/app/src/cli/utilities/app/config/webhooks.ts

```diff
@@ -111,3 +107,80 @@ export function validateInnerSubscriptions({endpoint, subscriptions = [], ...sch
     uniqueSubscriptionEndpointSet.add(key)
   }
 }
+
+export function transformWebhookConfig(content: object) {
+  const webhooks = getPathValue(content, 'webhooks') as WebhookConfig
+
+  // normalize webhook config with the top level config
+  const webhookSubscriptions = []
+  const {topics, subscriptions, endpoint} = webhooks
+
+  if (endpoint && topics?.length) {
```

Comment:
let's use `uri` since this is the term we're using in the TOML. WDYT?

Reply:
Yeah! I was planning to change both

Reply:
https://github.com/Shopify/shopify/pull/472090